src/braket/circuits/instruction.py

```diff
@@ -39,7 +39,8 @@ def __init__(self, operator: InstructionOperator, target: QubitSetInput):
 
         Raises:
             ValueError: If `operator` is empty or any integer in `target` does not meet the `Qubit`
-                or `QubitSet` class requirements.
+                or `QubitSet` class requirements. Also, if operator qubit count does not equal
```

Comment:
Also, if `operator`'s qubit count...

Reply:
Here, it's just using a noun as an adjective to describe qubit count.